### ISSUE TITLE
Example code doesn't work

### ISSUE THREAD

**Issue by th3b1gn0rth** [rfhb/ctrdata#18]
I am getting this:

Error: ctrGetQueryUrlFromBrowser(): 'url' and / or 'register' is not a single character string.

while following the example code

 ctrOpenSearchPagesInBrowser(input = "cancer&age=under-18",register="EUCTR")
 q <- ctrGetQueryUrlFromBrowser()
db <- nodbi::src_sqlite(
dbname = "sqlite_file.sql",
collection = "test"
)

ctrLoadQueryIntoDb(
queryterm = q,
only.count = TRUE,
con = db
)$n

I have windows 10 64 bit, R4.0.3.

**Comment by rfhb** [comment]
Please check, there should already be an error message following the command `q <- ctrGetQueryUrlFromBrowser()`. This error would mean that the clipboard does not have a register search URL, likely because the URL shown in the browser has not been copied into the clipboard. The user has to do this in this example, as follows: 

```
# 1.: execute
ctrOpenSearchPagesInBrowser(input = "cancer&age=under-18", register = "EUCTR")

# 2.: copy the URL from the browser's address bar to the clipboard
# Windows: select the address, Ctrl+A, Ctrl-C

# 3. execute
q <- ctrGetQueryUrlFromBrowser()
```

The [README](https://github.com/rfhb/ctrdata#example-workflow) for the example reads, "When trials of interest are listed in browser, copy the address from the browser’s address bar to the clipboard". 

Even though this issue is not a bug, I use the question to improve the error messaging in package `ctrdata` (should be in this repository over the next days).

**Comment by rfhb** [comment]
addressed with commit 4854f9bb817674b97507bf6004bf0bf0d37281a3 and commit cba8a1874ac6f1af8cb4b72b4c7b47c2514e0952
